auslib/admin/base.py

```diff
@@ -114,3 +134,22 @@ def add_security_headers(response):
 app.add_url_rule("/scheduled_changes/releases/<int:sc_id>/signoffs", view_func=ReleaseScheduledChangeSignoffsView.as_view("scheduled_change_release_signoffs"))
 app.add_url_rule("/scheduled_changes/releases/<int:sc_id>/revisions",
                  view_func=ReleaseScheduledChangeHistoryView.as_view("scheduled_change_releases_history"))
+app.add_url_rule("/scheduled_changes/required_signoffs/product", view_func=ProductRequiredSignoffsScheduledChangesView.as_view("scheduled_changes_product_rs"))
+app.add_url_rule("/scheduled_changes/required_signoffs/product/<int:sc_id>",
+                 view_func=ProductRequiredSignoffScheduledChangeView.as_view("scheduled_change_product_rs"))
+app.add_url_rule("/scheduled_changes/required_signoffs/product/<int:sc_id>/enact",
+                 view_func=EnactProductRequiredSignoffScheduledChangeView.as_view("enact_scheduled_change_product_rs"))
+app.add_url_rule("/scheduled_changes/required_signoffs/product/<int:sc_id>/signoffs",
+                 view_func=ProductRequiredSignoffScheduledChangeSignoffsView.as_view("scheduled_change_product_rs_signoffs"))
+app.add_url_rule("/scheduled_changes/required_signoffs/product/<int:sc_id>/revisions",
+                 view_func=ProductRequiredSignoffScheduledChangeHistoryView.as_view("scheduled_change_product_rs_history"))
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1340170 and https://bugzilla.mozilla.org/show_bug.cgi?id=1340171 for this.